app/buyers/views/buyers.py

```diff
@@ -80,163 +80,205 @@ def create_new_brief(framework_slug, lot_slug):
+    ]
+
+    return render_template(
+        "buyers/brief_overview.html",
+        confirm_remove=request.args.get("confirm_remove", None),
+        brief=brief,
+        sections=sections,
+        completed_sections=completed_sections,
+        step_sections=[section.step for section in sections if hasattr(section, 'step')],
+        delete_requested=delete_requested,
+    ), 200
+
+
+@buyers.route('/buyers/frameworks/<framework_slug>/requirements/<lot_slug>/<brief_id>/<section_slug>', methods=['GET'])
+def view_brief_section_summary(framework_slug, lot_slug, brief_id, section_slug):
+    get_framework_and_lot(framework_slug, lot_slug, data_api_client, status='live', must_allow_brief=True)
     brief = data_api_client.get_brief(brief_id)["briefs"]
-    if not is_brief_associated_with_user(brief, current_user.id) or not brief_can_be_edited(brief):
+
+    if not is_brief_correct(brief, framework_slug, lot_slug, current_user.id) or not brief_can_be_edited(brief):
         abort(404)
 
-    content = content_loader.get_manifest(framework_slug, 'edit_brief').filter(
-        {'lot': lot['slug']}
-    )
-    section = content.get_section(section_id)
-    if section is None or not section.editable:
+    content = content_loader.get_manifest(brief['frameworkSlug'], 'edit_brief').filter({'lot': brief['lotSlug']})
+    sections = content.summary(brief)
+    section = sections.get_section(section_slug)
+
+    if not section:
         abort(404)
 
     return render_template(
-        "buyers/edit_brief_section.html",
-        framework=framework,
-        data=brief,
+        "buyers/section_summary.html",
+        brief=brief,
         section=section
     ), 200
 
 
 @buyers.route(
-    '/buyers/frameworks/<framework_slug>/requirements/<lot_slug>/<brief_id>/edit/<section_id>',
-    methods=['POST'])
-def update_brief_submission(framework_slug, lot_slug, brief_id, section_id):
+    '/buyers/frameworks/<framework_slug>/requirements/<lot_slug>/<brief_id>/edit/<section_slug>/<question_id>',
+    methods=['GET'])
+def edit_brief_question(framework_slug, lot_slug, brief_id, section_slug, question_id):
+    get_framework_and_lot(framework_slug, lot_slug, data_api_client, status='live', must_allow_brief=True)
+    brief = data_api_client.get_brief(brief_id)["briefs"]
 
-    framework, lot = get_framework_and_lot(framework_slug, lot_slug, data_api_client,
-                                           status='live', must_allow_brief=True)
+    if not is_brief_correct(brief, framework_slug, lot_slug, current_user.id) or not brief_can_be_edited(brief):
+        abort(404)
+
+    content = content_loader.get_manifest(brief['frameworkSlug'], 'edit_brief').filter(
+        {'lot': brief['lotSlug']}
+    )
+    section = content.get_section(section_slug)
+    # https://github.com/alphagov/digitalmarketplace-supplier-frontend/blob/master/app/main/views/services.py#L381
+    # if section and (question_id is not None):
```

Comment:
That's what we do with multiquestions in the supplier app: we intercept any multiquestions in the view and turn them into section-like objects.
Here, we're unpacking multiquestions in the template file itself.

It would be nice to keep some of this as non-tacit knowledge, but, yeah, I don't know that this commented out stuff is the way to do it.